ci/Jenkinsfile

```diff
@@ -0,0 +1,148 @@
+pipeline {
+    agent any
+    options {
+        disableConcurrentBuilds(abortPrevious: true)
+    }
+    stages {
+        stage('Checks & Builds') {
+            failFast true
+            parallel {
+                stage('Check') {
+                    options {
+                        timeout(time: 1, unit: 'HOURS')
+                    }
+                    environment {
+                        HOME = "${WORKSPACE}"
+                    }
```

Comment:
Since this variable is required in all of the stages, it might be reasonable to move it to a top-level `environment` block. Underneath the `pipeline` directive.